www-apps/kibana-bin/kibana-bin-7.16.2.ebuild

```diff
@@ -81,7 +81,7 @@ src_install() {
 
 pkg_postinst() {
 	elog "This version of Kibana is compatible with Elasticsearch $(ver_cut 1-2) and"
-	elog "Node.js 14. Some plugins may fail with other versions of Node.js (Bug #656008)."
+	elog "Node.js 16. Some plugins may fail with other versions of Node.js (Bug #656008)."
```

Comment:
Sorry I'd missed the earlier PR for this. I'd again opened the tab and it got lost!

Reply:
No problem :)